### ISSUE TITLE
options.xhtml has a typo

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6417]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#6417|
|Type|defect|
|Created|2013-04-07 14:01:07Z|

`doc/core/howto/options.xhtml` has a spelling mistake: "occassion".

Attachments:

* [ticket_6417.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/5a9/5a925f457b67d9dc9a3366dac55f2b93e12f49e8/792cd4adf5a744208e3ee19657d54c7d4cd478ed.patch) (565 bytes) - added by jsnmoon on 2013-04-08 23:37:22Z - Fixes the spelling error.

<details><summary>Searchable metadata</summary>

```
trac-id__6417 6417
type__defect defect
reporter__itamar itamar
priority__lowest lowest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1365343267000000 1365343267000000
changetime__1365769810000000 1365769810000000
version__None None
owner__thijs thijs
cc__jsnmoon
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks for your patch, I'll apply it. Make sure to do supply patches that svn diff from the root of your checkout, see [TwistedDevelopment#SubmittingaPatch] for details.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [38044]) Apply ticket_6417.patch: Correct a typo in the usage.Options howto.

Author: jsnmoon
Reviewer: thijs
Fixes: [#6417](https://github.com/twisted/twisted/issues/6417)
